cuentociacua.txt

```diff
@@ -0,0 +1 @@
+Esta es la historia de dos niñas mimadas, prepotentes y fastidiosas que solo les gustaba hablar sobre lo que las otras niñas jugaban. Tanto lo hicieron que perdieron la alegriade vivir.
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.